main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -15,7 +15,7 @@
  */
 package ca.uwaterloo.flix.api
 
-import ca.uwaterloo.flix.api.Bootstrap.{EXT_CLASS, EXT_FPKG, EXT_JAR, FLIX_TOML, LICENSE, README, getArtifactDirectory, getManifestFile, getPkgFile}
+import ca.uwaterloo.flix.api.Bootstrap.{EXT_CLASS, EXT_FPKG, EXT_JAR, FLIX_TOML, LICENSE, README, getArtifactDirectory, getBuildDirectory, getManifestFile, getPkgFile}
```

Comment:
By the way, why are these imports here?

Reply:
They are defined in the `Bootstrap` object but not in the class, so in order to use them in the class definition without qualifying them, they have to be imported.

Reply:
Right, usually we qualify these things.